### ISSUE TITLE
Ребут фермы

### ISSUE THREAD

**Issue by Qiizoff** [kaseat/MinerWatchdog#3]
Было бы неплохо добавить возможность перегружать сам ПК! Бывает при майнинге на некоторых алгоритмах вылетают дрова на одной из карт и после перезагрузки майнера эта карта назодится в оффе. Помогает только ребут компа!

**Comment by CmRaD4u** [comment]
Поддерживаю. 
Либо это должно быть через реакцию на сообщение видюх 44 77 и тд по ошибкам.
Или можно вводить в каком-нибудь конфиг файле кол-во видеокарт, и если работает меньше, то выключать процесс майнинга и через паузу в сек(тоже добавить в конфиг) ребутить пк, можно добавить детект видюх раз в 15 минут, допустим.
Кстати было бы неплохо все же решить проблему с интернетом. 
Если его нет, то ставить все на паузу, если появляется то ребутить майнер.
А то при отключении инета скорость видюх становится по нулям и майнер начинает постоянно ребутиться, что в какой-то мере не есть правильно.

**Comment by zuma-rus** [comment]
Ещё очень нужно добавить параметр выбора GPU . Типа параметров:
--cuda-devices 0 1 2 3
--cuda_devices 0 1 2 3

ибо у меня стоит во всех майнерах параметр --cuda-devices 0 , но стоит ещё вторая видюха, которая просто стоит, чтобы монитор показывал и она у меня GPU 1
Программа же проверяет и видит две видюхи. GPU0 = 100%, а GPU1 = 1% и происходит постоянная перезагрузка майнера. 
Как объяснить программе, чтобы она GPU1 не учитывала, я так и не смог найти.

**Comment by kaseat** [comment]
Да, были уже такие предложения, скорее всего в следующем релизе добавлю.

**Comment by Qiizoff** [comment]
тоже возникла потребность в --cuda_devices! На Zec Miner 0.3.4b

**Comment by Qiizoff** [comment]
похоже что разрабы забили на проект!

**Comment by kaseat** [comment]
Нет, проект не забыт. В скором времени планируется обновление.
